src/data/analyse_en.json

```diff
@@ -78,10 +78,10 @@
             "totalKey": ["app_downloads_cumulated"]
         },
         {
-            "title": "Transmitted and registered test results",
+            "title": "Provided test results",
             "export": "https://obs.eu-de.otc.t-systems.com/obs-public-dashboard/csv/test_results_data.zip",
             "info": {
-                "content": "<strong>Transmitted and registered test results</strong><p>Count of PCR test results and antigen rapid test results submitted by laboratories and registered in the Corona-Warn-App, as well as the sum of test results of both test types.</p><strong>Transmitted and registered test results: PCR tests</strong><p>Count of positive, negative and invalid as well as the sum of all PCR test results submitted by laboratories and registered in the Corona-Warn-App.</p><strong>Transmitted and registered test results: Rapid antigen tests</strong><p>Count of positive, negative and invalid as well as the sum of all rapid antigen test results submitted by laboratories and registered in the Corona-Warn-App.</p><strong>Time intervals</strong><p>Daily: Count accurate to the day<br />Weekly: Weekly count by calendar week (CW)<br />7-day moving average: 7-day moving average accurate to the day<br />Cumulative: Cumulative sum accurate to the day, since the beginning of the measurement</p><strong> Reference date</strong><p>The reference date is the day of the test submission to the test server.</p><strong> Data sources </strong><p>Corona-Warn-App backend</p>"
+                "content": "<strong>Provided test results</strong><p>Count of PCR test results and antigen rapid test results provided by laboratories and rapid test sites on the Corona-Warn-App server, as well as the sum of test results of both test types.</p><strong>Provided test results: PCR tests</strong><p>Count of positive, negative and invalid as well as the sum of all PCR test results provided by laboratories and rapid test sites on the Corona-Warn-App server.</p><strong>Provided test results: Rapid antigen tests</strong><p>Count of positive, negative and invalid as well as the sum of all rapid antigen test results provided by laboratories and rapid test sites on the Corona-Warn-App server.</p><strong>Time intervals</strong><p>Daily: Count accurate to the day<br />Weekly: Weekly count by calendar week (CW)<br />7-day moving average: 7-day moving average accurate to the day<br />Cumulative: Cumulative sum accurate to the day, since the beginning of the measurement</p><strong> Reference date</strong><p>The reference date is the day of the test submission to the test server.</p><strong> Data sources </strong><p>Corona-Warn-App backend</p>"
```

Comment:
See above also if you decide to change the word order in the title, then also in the title of the text here i.e. "Test results provided"